flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -98,24 +98,24 @@ class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
   final bool autoInitialize;
 
   /// Only set for [asset] videos. The package that the asset was loaded from.
-  String package;
+  String? package;
 
   /// Describes the type of data source this [VlcPlayerController]
   /// is constructed with.
   DataSourceType get dataSourceType => _dataSourceType;
   DataSourceType _dataSourceType;
 
   /// Determine if platform is ready to call initialize method
-  bool get isReadyToInitialize => _isReadyToInitialize;
-  bool _isReadyToInitialize;
+  bool? get isReadyToInitialize => _isReadyToInitialize;
+  bool? _isReadyToInitialize;
 
   /// This is just exposed for testing. It shouldn't be used by anyone depending
   /// on the plugin.
   @visibleForTesting
-  int get viewId => _viewId;
+  int? get viewId => _viewId;
 
   /// The viewId for this controller
-  int _viewId;
+  int? _viewId;
```

Comment:
Not really sure this has to be nullable.
In any case the code below force-unwraps this variable, and will cause a runtime exception if it happens to be null.

We could:
1. Add a non-null assertion to the getter above, and return `int` instead of `int?`.
2. Make the variable `late final`.

I'm personally more in favour of option 1, since it feels more runtime-safe.

Reply:
Went with late init since the getter is used for testing only. The _viewId comes from the platform itself via 

Future<void> onPlatformViewCreated(int viewId) async